src/NuGetTrends.Api/ExceptionInResponseMiddleware.cs

```diff
@@ -7,6 +7,7 @@
 
 namespace NuGetTrends.Api
 {
+    // ReSharper disable once ClassNeverInstantiated.Global - reflection
```

Comment:
I think we should switch to using the [Exception Handler Middleware](https://docs.microsoft.com/en-us/aspnet/core/web-api/handle-errors?view=aspnetcore-3.0#exception-handler) in favor of this. Also, maybe we want to add [Exception Filters](https://docs.microsoft.com/en-us/aspnet/core/mvc/controllers/filters?view=aspnetcore-3.0#exception-filters), to handle exceptions from inside the MVC pipeline. Maybe even using the model `ProblemDetails` to make it consistent?

Reply:
Thinking better, this doesn't belong in this PR. I'll open a ticket so we can work on it separately.

Reply:
OK, please raise this (I cant' as I don't know what you mean :)